examples/twittermap/web/app/views/twittermap/index.scala.html

```diff
@@ -25,6 +25,15 @@
    <search-bar></search-bar>
    <predefined-keywords></predefined-keywords>
 
+  <!-- modifiy hide and show button -->
+
+  <!-- <button type="button" class="slide-up-down">down</button> -->
```

Comment:
Let's remove useless comments. Only keep comment for current implementation of this button.